src/jit/codegenxarch.cpp

```diff
@@ -4135,22 +4193,22 @@ void CodeGen::genCodeForCpObj(GenTreeObj* cpObjNode)
         {
 #ifdef DEBUG
             // If the destination of the CpObj is on the stack
-            // make sure we allocated RCX to emit rep movs{d,q}.
+            // make sure we allocated RCX to emit rep movsp.
```

Comment:
OK - how about "make sure we allocated RCX to emit the movsp (alias for movsd or movsq for 32 and 64 bits respectively)"?
That way the reader won't be confused about why we're talking about movs{d,q} in the comment and movsp in the code.

---
In reply to: [79508080](https://github.com/dotnet/coreclr/pull/7252#discussion_r79508080) [](ancestors = 79508080)